docker/files/run/run.sh

```diff
@@ -41,40 +72,30 @@ function apply_configuration() {
   else
     diagcfg telemetry disable
   fi
-}
 
-function catchup() {
-  local FAST_CATCHUP_URL="https://algorand-catchpoints.s3.us-east-2.amazonaws.com/channel/CHANNEL/latest.catchpoint"
-  local CATCHPOINT=$(curl -s ${FAST_CATCHUP_URL/CHANNEL/$NETWORK})
-  if [[ "$(echo $CATCHPOINT | wc -l | tr -d ' ')" != "1" ]]; then
-    echo "Problem starting fast catchup."
-    exit 1
+  # start kmd
+  if [ "$START_KMD" = "1" ]; then
+    local KMD_DIR="kmd-v0.5"
+    # on intial bootstrap, this directory won't exist.
+    mkdir -p "$KMD_DIR"
+    if [ -f "/etc/kmd_config.json" ]; then
+      cp /etc/kmd_config.json kmd_config.json
+    else
+      echo "{ \"address\":\"0.0.0.0:${KMD_PORT}\", \"allowed_origins\":[\"*\"] }" >kmd_config.json
+    fi
+
+    if [ "$KMD_TOKEN" = "" ] && [ "$ADMIN_TOKEN" != "" ]; then
+      echo "$ADMIN_TOKEN" >kmd.token
+    else
+      echo "$KMD_TOKEN" >kmd.token
+    fi
   fi
-
-  sleep 5
-  goal node catchup "$CATCHPOINT"
 }
 
-function start_public_network() {
-  cd "$ALGORAND_DATA"
-
-  apply_configuration
-
-  if [[ $FAST_CATCHUP ]]; then
-    catchup &
+function start_kmd() {
+  if [ "$START_KMD" = "1" ]; then
+    kmd start -d "$ALGORAND_DATA" -t 0
```

Comment:
I just noticed that sandbox uses goal to start KMD:
```suggestion
    goal kmd start -d "$ALGORAND_DATA"
```